modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/coldstorage/MoveToColdStorage.java

```diff
@@ -50,13 +57,13 @@
     @Param(name = "save", required = false, values = "true")
     protected boolean save = true;
 
-    @OperationMethod(collector = DocumentModelCollector.class)
-    public DocumentModel run(DocumentModel doc) {
+    @OperationMethod
+    public DocumentModel run(DocumentModel document) {
         // retrieve the thumbnail which will be used to replace the content, once the move done
-        Blob thumbnail = Framework.getService(ThumbnailService.class).getThumbnail(doc, session);
+        Blob thumbnail = Framework.getService(ThumbnailService.class).getThumbnail(document, session);
```

Comment:
We use `doc` in lots of places to refer to `DocumentModel` instances, not sure why it's worth changing here.